decidim-core/db/migrate/20190412131728_fix_user_names.rb

```diff
@@ -8,14 +8,15 @@ def change
     characters_to_remove = "<>?%&^*\#@()[]=+:;\"{}\\|/"
 
     weird_characters.each do |character|
-      Decidim::UserBaseEntity.where("name like '%#{character}%' escape '\' OR nickname like '%#{character}%' escape '\'").find_each do |entity|
-        Rails.logger.log "detected character: #{character}"
-        Rails.logger.log "UserBaseEntity ID: #{entity.id}"
-        Rails.logger.log "#{entity.name} => #{entity.name.delete(characters_to_remove).strip}"
-        Rails.logger.log "#{entity.nickname} => #{entity.nickname.delete(characters_to_remove).strip}"
+      Decidim::UserBaseEntity.not_deleted.where("name like '%#{character}%' escape '\' OR nickname like '%#{character}%' escape '\'").find_each do |entity|
```

Comment:
@oriolgual are you sure that `not_deleted` scope exists for `Decidim::UserBaseEntity` ?
I've checked `Decidim::UserBaseEntity.not_deleted` in rails console, and it returns ```(undefined method 'not_deleted' for #<Class:0x0000564064b3c4c8>)```

Reply:
Ohhh, you're right!